### ISSUE TITLE
How to create shuffle 

### ISSUE THREAD

**Issue by Zexeta** [OpenKore/openkore#2425]
I am trying to update it for restart myself since whoever was doing it doesn't seem to anymore. I followed the guide that was posted it looks like I got the others files but peek does not work with the client. Is there another way to create the shuffle file?

**Comment by alisonrag** [comment]
are you sure? does 'peek' not work or you not know how to convert output file to work in openkore?

![image](https://user-images.githubusercontent.com/10372732/47238448-a4f89d80-d3b8-11e8-99ce-35b47e774615.png)

**Comment by Zexeta** [comment]
Peek won't even open the files properly for me and im not sure why.  Says Reference Location not found and does nothing for me.

**Comment by alisonrag** [comment]
step by step:
1 - Update iRO
2 - Download and extract peek: https://gitlab.com/4144/peek/
3 - Execute Peek as administrator
4 - Select "Ragexe.exe" in iRO folder
5 - Click in "Extract Lengths"
6 - Wait a minute
7 - Go to peek\output\
8 - Search for a file "PacketLengths_YYYYMMDD.ini"

inside this file have:
recvpackets
shuffle packets
cryptkeys
sync packets

just need convert to openkore

**Comment by Zexeta** [comment]
Still wouldn't work on this pc, but worked on the other one fine.  Any info out there on how to convert it?

**Comment by alisonrag** [comment]
Use this file as base:
https://github.com/OpenKore/openkore/blob/master/tables/kRO/Sakray/shuffle.txt
Keep the left packet and replace the right 
Example:
Peek:
```
1: 0x0ABD
2: 0x0433
``` 

Shuffle
```
0089 0437 # actor_action
0113 0438 # skill_use
```

The result will be
```
0089 0ABD
0113 0433
```

**Comment by Zexeta** [comment]
Hmm ok i will try it, i looked and assumed it was something like that but when comparing last weeks shuffle this this weeks and peek they didn't seem to even fall in the same order.

**Comment by Zexeta** [comment]
That didn't work, the output from peek actually has more packets than the shuffle file.

**Comment by Skye168** [comment]
I guess I may have this figure it out for each packet for Restart but what about the other additional packet in this shuffle.txt, There must be some easier way to identify which to delete or clean up. 

# Generated: 2018-10-14T06:49:10Z 
# iRO Restart shuffle.txt packets based from gudoh last fix, replaced with new packet from peek one by one.

007D 091F # actor_action
0090 0930 # skill_use
00B8 0955 # character_move
0126 0945 # sync
0127 0957 # actor_look_at
0140 0941 # item_take
0178 0881 # item_drop
014D 0817 # storage_item_add
014F 0920 # storage_item_remove

0AF1 0821 

08C9 086C # skill_use_location
09CE 0202 # not implemented in openkore (CZ_USE_SKILL_TOGROUND_WITHTALKBOX)
0A68 089D # actor_name_request
0A5A 086B # actor_info_request
0A5C 0948 # not implemented in openkore (CZ_SSILIST_ITEM_CLICK)
0A69 0369 # search_store_request_next_page
0A6C 0969 # search_store_info

0A6E 08C9
0A0D 0A8B
0AF1 0821
0AFD 0B0C

0437 07EC # buy_bulk_buyer
0438 0964 # buy_bulk_request
035F 0A6E # buy_bulk_closeShop
0360 0946 # buy_bulk_openShop
0361 0934 # booking_register
0362 0926 # not implemented in openkore (CZ_JOIN_BATTLE_FIELD)
0363 09CE # item_list_window_selected
0364 0886 # map_login

0365 092F # party_join_request_by_name
0366 08A4 # not implemented in openkore (CZ_GANGSI_RANK)
0367 0360 # friend_request
0368 089C # homunculus_command
0369 0894 # storage_password

083C 0838
0838 0835
0835 0819
0819 0A6E
0817 0360
0815 0817
0811 0178
0802 007D
07EC 0363
07E4 0281
0436 0090
02C4 0802
0281 0436
0202 02C4
022D 00B8
023B 087E
087E 023B
0918 0940
0919 0918
093B 0361
0940 0202
0962 0A5C

**Comment by alisonrag** [comment]
You can delete the rest of packets and use only from the file that i mentioned

**Comment by alisonrag** [comment]
I will creste a tutorial and script afte that i will close here

**Comment by c4c1n6kr3m1** [comment]
57 my azz, it's 213 if u use allanons
they taught u , and still u ....swt
007D 0437
0437 091F
091F 0363
0363 0881
0881 093B
093B 0965
0965 085F
085F 088A
088A 093A
093A 093E
093E 0933
0933 0864
0864 0953
0953 089B
089B 0861
0861 092E
092E 085A
085A 0943
0943 0895
0895 0897
0897 0838
0838 0369
0369 086B
086B 007D
0360 0945
0945 0875
0875 096A
096A 088D
088D 095D
095D 0815
0815 0A6E
0A6E 0878
0878 094E
094E 088C
088C 0927
0927 0936
0936 094B
094B 08A9
08A9 0940
0940 087F
087F 0888
0888 0917
0917 0871
0871 089E
089E 0924
0924 0364
0364 0817
0817 0964
0964 0877
0877 0956
0956 093C
093C 0960
0960 0A68
0A68 086A
086A 088F
088F 08A5
08A5 02C4
02C4 092F
092F 0A6C
0A6C 0931
0931 0892
0892 085D
085D 0A5C
0A5C 0876
0876 095A
095A 0879
0879 091E
091E 0944
0944 0899
0899 0952
0952 093D
093D 092B
092B 0887
0887 0367
0367 0202
0202 0360
091A 0932
0932 0898
0898 0436
0436 0886
0886 0951
0951 0281
0281 08A4
08A4 08A1
08A1 094F
094F 0A5A
0A5A 0880
0880 0949
0949 0925
0925 0869
0869 087D
087D 093F
093F 0889
0889 092D
092D 0362
0362 0941
0941 0863
0863 094C
094C 094A
094A 0922
0922 0868
0868 0126
0126 08A0
08A0 0921
0921 08AD
08AD 08A8
08A8 00B8
00B8 035F
035F 0955
0955 0947
0947 087A
087A 0140
0140 0958
0958 0967
0967 0874
0874 0963
0963 089F
089F 0811
0811 0946
0946 086D
086D 0872
0872 0885
0885 08AA
08AA 095F
095F 085B
085B 0891
0891 087E
087E 091C
091C 0438
0438 0930
0930 0939
0939 0366
0366 086C
086C 0929
0929 0127
0127 0361
0361 0957
0957 086F
086F 0938
0938 0961
0961 08A2
08A2 08A6
08A6 08C9
08C9 0923
0923 022D
022D 089C
089C 0928
0928 085E
085E 08A3
08A3 0882
0882 014D
014D 0365
0365 0920
0920 0883
0883 0802
0802 0934
0934 0890
0890 0090
0090 091B
091B 092A
092A 0819
0819 07EC
07EC 0926
0926 0860
0860 0893
0893 089A
089A 0368
0368 089D
089D 0950
0950 014F
014F 083C
083C 0948
0948 0962
0962 092C
092C 0919
0919 08A7
08A7 094D
094D 08AC
08AC 0896
0896 0942
0942 0968
0968 087C
087C 0867
0867 095E
095E 0866
0866 088B
088B 095C
095C 07E4
07E4 09CE
09CE 0966
0966 0835
0835 0969
0969 091A
0862 085C
085C 0959
0959 088E
088E 091D
091D 0865
0865 0870
0870 0862
0873 08AB
08AB 0884
0884 086E
086E 0935
0935 095B
095B 087B
087B 0918
0918 0954
0954 0A69
0A69 0873
023B 0894
0894 0937
0937 023B

or learn what alisonrag wrote
i used my eyes and msword before, yes , msword

**Comment by Zexeta** [comment]
Is this previous post supposed to be correct? My not no longer even makes it to char select now? It did with what I sorted out but now does nothing.

**Comment by c4c1n6kr3m1** [comment]
char select? it has nothing todo with shuffle
shuffled packets only after in map
read those 28 packets alisonrag wrote

**Comment by Zexeta** [comment]
> char select? it has nothing todo with shuffle
> shuffled packets only after in map
> read those 28 packets alisonrag wrote

Managed to get in game but its crashing when switching gear and doesn't seem to be trigger useselfskill blocks n such?

**Comment by c4c1n6kr3m1** [comment]
i said it has nothing todo with shuffle.txt
```
Party EXP set to Even Share
Party item set to Even Share
Party item division set to Individual Take
You unequip Novice Adventurer's Suit [Roda Frog] [1] (58) - Armor
You equip Novice Adventurer's Suit [1] (56) - Armor (type 16)
NPC Eden Teleport Officer#2 (2) shown minimap indicator at location 161, 97 with the color [R:255, G:128, B:0, A:0]
move moc_fild12
Calculating route to: Sograt Desert(moc_fild12)
NPC Exists: Trader (155, 69) (ID 50063) - (0)
NPC Exists: Jeweler (166, 51) (ID 50057) - (1)
NPC Exists: Jeweler (146, 43) (ID 50058) - (2)
Portal Exists: morocc -> moc_fild12 (160, 17) - (0)
Portal Exists: morocc -> moc_fild12 (160, 17) - (1)
Map Change: moc_fild12.gat (159, 378)
Portal Exists: moc_fild12 -> morocc (159, 381) - (0)
NPC Exists: majabara (160, 372) (ID 50008) - (0)
NPC Exists: Ashe#3 (160, 365) (ID 50009) - (1)
NPC Exists: Trace of Battle#1 (166, 369) (ID 50031) - (2)
You unequip Novice Adventurer's Suit [1] (58) - Armor
You equip Novice Adventurer's Suit [Roda Frog] [1] (56) - Armor (type 16)
```

**Comment by Skye168** [comment]
Thank you. It works perfectly. Where can I learn how do you get this shuffer.txt? 
Are there any tutorial?
Thanks.

**Comment by Zexeta** [comment]
> i said it has nothing todo with shuffle.txt
> 
> ```
> Party EXP set to Even Share
> Party item set to Even Share
> Party item division set to Individual Take
> You unequip Novice Adventurer's Suit [Roda Frog] [1] (58) - Armor
> You equip Novice Adventurer's Suit [1] (56) - Armor (type 16)
> NPC Eden Teleport Officer#2 (2) shown minimap indicator at location 161, 97 with the color [R:255, G:128, B:0, A:0]
> move moc_fild12
> Calculating route to: Sograt Desert(moc_fild12)
> NPC Exists: Trader (155, 69) (ID 50063) - (0)
> NPC Exists: Jeweler (166, 51) (ID 50057) - (1)
> NPC Exists: Jeweler (146, 43) (ID 50058) - (2)
> Portal Exists: morocc -> moc_fild12 (160, 17) - (0)
> Portal Exists: morocc -> moc_fild12 (160, 17) - (1)
> Map Change: moc_fild12.gat (159, 378)
> Portal Exists: moc_fild12 -> morocc (159, 381) - (0)
> NPC Exists: majabara (160, 372) (ID 50008) - (0)
> NPC Exists: Ashe#3 (160, 365) (ID 50009) - (1)
> NPC Exists: Trace of Battle#1 (166, 369) (ID 50031) - (2)
> You unequip Novice Adventurer's Suit [1] (58) - Armor
> You equip Novice Adventurer's Suit [Roda Frog] [1] (56) - Armor (type 16)
> ```

Are you in xkore mode?

**Comment by Skye168** [comment]
Thanks, I'll take a look & follow step by step after this to see if I could get the same result. 
This is very good information. Appreciate it.

**Comment by Zexeta** [comment]
I took last weeks shuffle file, replaced the right values with the ones from Peek, i got this, last weeks on left, right is with the values from peek.
https://i.imgur.com/t9q3TM2.png
It doesn't even login with this setup though.  I was under the impression last week extra packets were added to the shuffle or something people were saying.  From what i understand this should work since alison said to just delete the others.  I tested the guys that was 200+ to see where i went wrong and the bot can't view equippable items and crashes when trying to equip them through the client but does connect.  He posted to replace the packet that popped up in the error with the one from peek, which was already done in the shuffle prior to the crash.  

People are posting all sorts of different things.  Alison is posting a kRO one as a baseline that doesn't match restarts, so i use last weeks restart one as a baseline, another guy is posting one with literally every packet 200+ shuffled so i changed the values on the right chronologically and it still doesn't work.

**Comment by c4c1n6kr3m1** [comment]
i don't regularly updgraded my openkore, mine is years behind because of too many old macros not compatible
but this packet not list on iro
try added on send\iRO.pm if not exist

'0438' => ['skill_use', 'v2 a4', [qw(lv skillID targetID)]],

skill_use 0438

**Comment by ellenreval** [comment]
I did peek as what alisonrag says... 
how can i convert it to openkore..
and how about the server.txt
[Packet_Lengths]
0x091F = 7
0x0930 = 10
0x0955 = 5
0x0945 = 6
0x0957 = 5
0x0941 = 6
0x0881 = 6
0x0817 = 8
0x0920 = 8
0x086C = 10
0x0202 = 90
0x089D = 6
0x086B = 6
0x0948 = 12
0x0369 = 2
0x0969 = -1
0x07EC = -1
0x0964 = 6
0x0A6E = 2
0x0946 = -1
0x0934 = 18
0x0926 = 8
0x09CE = -1
0x0886 = 19
0x092F = 26
0x08A4 = 4
0x0360 = 26
0x089C = 5
0x0894 = 36
0x088D = 2
0x0943 = 2
0x0891 = 2
0x0959 = 2
0x0A5C = 2
0x08A3 = 2
0x088A = 2
0x0893 = 2
0x092E = 2
0x085C = 2
0x094C = 2
0x0953 = 2
0x0870 = 2
0x088B = 2
0x095E = 2
0x0126 = 2
0x087D = 2
0x088F = 2
0x007D = 2
0x0929 = 2
0x0872 = 2
0x0935 = 2
0x0938 = 2
0x0862 = 2
0x089E = 2
0x0885 = 2
0x08AB = 2
0x0963 = 2
0x096A = 2
0x095A = 2
0x0956 = 2
0x094E = 2
0x091E = 2
0x0140 = 2
0x0918 = 2
0x0867 = 2
0x093F = 2
0x091C = 2
0x0888 = 2
0x0949 = 2
0x093B = 2
0x014D = 2
0x0802 = 2
0x0871 = 2
0x0954 = 2
0x08A7 = 2
0x0932 = 2
0x092A = 2
0x0438 = 2
0x0865 = 2
0x0944 = 2
0x0363 = 2
0x0883 = 2
0x08AD = 2
0x0868 = 2
0x022D = 2
0x0364 = 2
0x0869 = 2
0x0860 = 2
0x0936 = 2
0x085E = 2
0x0127 = 2
0x0819 = 2
0x0887 = 2
0x0919 = 2
0x0362 = 2
0x085A = 2
0x0A6C = 2
0x0939 = 2
0x0892 = 2
0x0898 = 2
0x0864 = 2
0x0890 = 2
0x095B = 2
0x094B = 2
0x023B = 2
0x0961 = 2
0x0366 = 2
0x093E = 2
0x0965 = 2
0x0960 = 2
0x092B = 2
0x0933 = 2
0x0889 = 2
0x087F = 2
0x086E = 2
0x08AA = 2
0x0951 = 2
0x0367 = 2
0x0917 = 2
0x092D = 2
0x093A = 2
0x095C = 2
0x0927 = 2
0x095D = 2
0x091D = 2
0x08A5 = 2
0x0090 = 2
0x087E = 2
0x085D = 2
0x089A = 2
0x0937 = 2
0x0897 = 2
0x0942 = 2
0x0838 = 2
0x0436 = 2
0x0952 = 2
0x0368 = 2
0x0861 = 2
0x0928 = 2
0x0950 = 2
0x0924 = 2
0x0811 = 2
0x0921 = 2
0x094F = 2
0x08A6 = 2
0x0882 = 2
0x08A1 = 2
0x02C4 = 2
0x08C9 = 2
0x094D = 2
0x00B8 = 2
0x0940 = 2
0x095F = 2
0x0884 = 2
0x0896 = 2
0x08A8 = 2
0x0863 = 2
0x0968 = 2
0x0895 = 2
0x0899 = 2
0x0875 = 2
0x086D = 2
0x087A = 2
0x0962 = 2
0x0925 = 2
0x0922 = 2
0x08A9 = 2
0x094A = 2
0x08AC = 2
0x088C = 2
0x0A5A = 2
0x014F = 2
0x0281 = 2
0x093D = 2
0x089B = 2
0x0A69 = 2
0x0947 = 2
0x093C = 2
0x086F = 2
0x0967 = 2
0x088E = 2
0x0879 = 2
0x087B = 2
0x07E4 = 2
0x0815 = 2
0x0866 = 2
0x085B = 2
0x0A68 = 2
0x08A2 = 2
0x092C = 2
0x089F = 2
0x0877 = 2
0x085F = 2
0x0835 = 2
0x0874 = 2
0x087C = 2
0x091A = 2
0x0187 = 6
0x0081 = 3
0x01C6 = 4
0x01C7 = 2
0x0064 = 55
0x0069 = -1
0x006A = 23
0x01DB = 2
0x01DC = -1
0x01DD = 47
0x01FA = 48
0x0204 = 18
0x01F1 = -1
0x0200 = 26
0x01BE = 2
0x01BF = 3
0x0065 = 17
0x0066 = 3
0x0067 = 37
0x0970 = 31
0x020D = -1
0x006B = -1
0x006C = 3
0x006D = 147
0x006E = 3
0x0071 = 28
0x0068 = 46
0x01FB = 56
0x006F = 2
0x0070 = 3
0x02CA = 3
0x0072 = 19
0x0082 = 2
0x0085 = 5
0x0089 = 7
0x008C = -1
0x007E = 6
0x0437 = 2
0x091B = 7
0x0099 = -1
0x019C = -1
0x009B = 5
0x00CC = 6
0x00CE = 2
0x009F = 6
0x00A2 = 6
0x00A7 = 8
0x00A9 = 6
0x00AB = 4
0x035F = 7
0x00B9 = 6
0x00B2 = 3
0x00BA = 2
0x00BB = 5
0x00BF = 3
0x00C1 = 2
0x00C5 = 7
0x00C8 = -1
0x00C9 = -1
0x0096 = -1
0x00CF = 27
0x00D0 = 3
0x00D3 = 2
0x00D5 = -1
0x00D9 = 14
0x00DE = -1
0x00E0 = 30
0x00E2 = 26
0x00E3 = 2
0x00E4 = 6
0x00E6 = 3
0x00E8 = 8
0x00EB = 2
0x00ED = 2
0x00EF = 2
0x00F3 = 8
0x00F5 = 8
0x00F7 = 2
0x00F9 = 26
0x01E8 = 28
0x00FC = 6
0x00FF = 10
0x0102 = 6
0x0100 = 2
0x0103 = 30
0x0108 = -1
0x0112 = 4
0x0113 = 10
0x0116 = 10
0x0118 = 2
0x011B = 20
0x011D = 2
0x08A0 = 8
0x0361 = 8
0x0128 = 8
0x0129 = 8
0x012A = 2
0x012E = 2
0x012F = -1
0x01B2 = -1
0x0134 = -1
0x0130 = 6
0x0138 = 3
0x013F = 26
0x0958 = 22
0x0143 = 10
0x0146 = 6
0x0178 = 4
0x017A = 4
0x017C = 6
0x01FD = 15
0x018A = 4
0x018E = 10
0x0190 = 90
0x0197 = 4
0x0198 = 8
0x01AE = 4
0x025B = 6
0x01AF = 4
0x01B9 = 6
0x01CD = 30
0x01CE = 6
0x0443 = 8
0x01CF = 28
0x01D0 = 8
0x01E1 = 8
0x01D1 = 14
0x01D2 = 10
0x0094 = 6
0x0095 = 30
0x0195 = 102
0x0193 = 6
0x0175 = 6
0x0176 = 106
0x0079 = 53
0x019D = 6
0x014C = -1
0x0365 = 2
0x014E = 6
0x083C = 6
0x0150 = 110
0x01B6 = 114
0x0151 = 6
0x0152 = -1
0x0153 = -1
0x0154 = -1
0x0166 = -1
0x0155 = -1
0x0156 = -1
0x0157 = 6
0x0159 = 54
0x015A = 66
0x015B = 54
0x015C = 90
0x015D = 42
0x015E = 6
0x015F = 42
0x0160 = -1
0x0161 = -1
0x0162 = -1
0x0163 = -1
0x0164 = -1
0x0165 = 30
0x0167 = 3
0x0168 = 14
0x0169 = 3
0x016A = 30
0x016B = 10
0x0149 = 9
0x014A = 6
0x014B = 27
0x016E = 186
0x016F = 182
0x017E = -1
0x017F = -1
0x016C = 43
0x016D = 14
0x01F2 = 20
0x0170 = 14
0x0171 = 30
0x0172 = 10
0x0173 = 3
0x0174 = -1
0x0180 = 6
0x0181 = 3
0x0182 = 106
0x0183 = 10
0x0184 = 10
0x0185 = 34
0x019E = 2
0x019F = 6
0x01A0 = 3
0x01A1 = 3
0x01A2 = 37
0x01A3 = 5
0x01A4 = 11
0x01A5 = 26
0x01A6 = -1
0x01A7 = 4
0x01A8 = 4
0x01A9 = 6
0x01AA = 10
0x01CA = 3
0x01B0 = 11
0x01B1 = 7
0x01BA = 26
0x01BB = 26
0x01BC = 26
0x01BD = 26
0x01C0 = 2
0x01C1 = 14
0x01C2 = 10
0x01D3 = 35
0x01D5 = -1
0x01D4 = 6
0x01DF = 6
0x01F3 = 10
0x0284 = 14
0x01FF = 10
0x01ED = 2
0x01E7 = 2
0x01B7 = 6
0x01F7 = 14
0x01E3 = 14
0x01CB = 9
0x01F9 = 6
0x01E5 = 6
0x0201 = -1
0x0203 = 10
0x0205 = 26
0x0206 = 11
0x0207 = 34
0x0208 = 14
0x0209 = 36
0x020A = 10
0x020E = 32
0x0212 = 26
0x0213 = 26
0x0214 = 42
0x0215 = 6
0x0216 = 6
0x0217 = 2
0x0218 = 2
0x0225 = 2
0x0219 = 282
0x021A = 282
0x0226 = 282
0x0282 = 284
0x021B = 10
0x021C = 10
0x0224 = 10
0x0280 = 12
0x0285 = 6
0x0286 = 4
0x021D = 6
0x021E = 6
0x021F = 66
0x0222 = 6
0x0221 = -1
0x0220 = 10
0x0223 = 8
0x0073 = 11
0x0074 = 3
0x0075 = -1
0x0076 = 9
0x0077 = 5
0x0078 = 55
0x007A = 58
0x007B = 60
0x007C = 44
0x007F = 6
0x0080 = 7
0x0083 = 2
0x0084 = 2
0x0086 = 16
0x0087 = 12
0x0088 = 10
0x08CD = 10
0x008A = 29
0x008B = 23
0x008D = -1
0x008E = -1
0x0091 = 22
0x0092 = 28
0x0093 = 2
0x0097 = -1
0x0098 = 3
0x009A = -1
0x009C = 9
0x009D = 17
0x009E = 17
0x00A0 = 23
0x00A1 = 6
0x00A3 = -1
0x00A4 = -1
0x00A5 = -1
0x00A6 = -1
0x00A8 = 7
0x00AA = 9
0x00AC = 7
0x00AE = -1
0x00AF = 6
0x00B0 = 8
0x00B1 = 8
0x00B3 = 3
0x00B4 = -1
0x00B5 = 6
0x00B6 = 6
0x00B7 = -1
0x00BC = 6
0x00BD = 44
0x00BE = 5
0x00C0 = 7
0x00C2 = 6
0x00C3 = 8
0x00C4 = 6
0x00C6 = -1
0x00C7 = -1
0x00CA = 3
0x00CB = 3
0x00CD = 3
0x00D1 = 4
0x00D2 = 4
0x00D4 = -1
0x00D6 = 3
0x00D7 = -1
0x00D8 = 6
0x00DA = 3
0x00DB = -1
0x00DC = 28
0x00DD = 29
0x00DF = -1
0x00E1 = 30
0x00E5 = 26
0x00E7 = 3
0x00E9 = 19
0x00EA = 5
0x00EC = 3
0x00EE = 2
0x00F0 = 3
0x00F1 = 2
0x00F2 = 6
0x00F4 = 21
0x00F6 = 8
0x00F8 = 2
0x00FA = 3
0x00FB = -1
0x00FD = 27
0x00FE = 30
0x0101 = 6
0x0104 = 79
0x0105 = 31
0x0106 = 10
0x0107 = 10
0x0109 = -1
0x010A = 4
0x010B = 6
0x010C = 6
0x010D = 2
0x010E = 11
0x010F = -1
0x02B1 = -1
0x02B2 = -1
0x02B5 = -1
0x0110 = 10
0x0111 = 39
0x0114 = 31
0x0115 = 35
0x0117 = 18
0x0119 = 13
0x0229 = 15
0x011A = 15
0x011C = 68
0x011E = 3
0x011F = 16
0x0120 = 6
0x0121 = 14
0x0122 = -1
0x0123 = -1
0x0124 = 21
0x0125 = 8
0x012B = 2
0x012C = 3
0x012D = 4
0x0131 = 86
0x0132 = 6
0x0133 = -1
0x0135 = 7
0x0136 = -1
0x0137 = 6
0x0139 = 16
0x013A = 4
0x013B = 4
0x013C = 4
0x013D = 6
0x013E = 24
0x0141 = 14
0x0142 = 6
0x0144 = 23
0x0145 = 19
0x0147 = 39
0x0148 = 8
0x0177 = -1
0x0179 = 5
0x017B = -1
0x017D = 7
0x0188 = 8
0x0189 = 4
0x018B = 4
0x018C = 29
0x018D = -1
0x018F = 6
0x0191 = 86
0x0192 = 24
0x0194 = 30
0x0196 = 9
0x028A = 18
0x0199 = 4
0x019A = 14
0x019B = 10
0x01AB = 12
0x01AC = 6
0x01AD = -1
0x025A = -1
0x01B3 = 67
0x01B4 = 12
0x01B5 = 18
0x01B8 = 3
0x01C3 = -1
0x01C4 = 22
0x01C5 = 22
0x01C8 = 13
0x01C9 = 97
0x01CC = 9
0x01D6 = 4
0x01D7 = 11
0x01D8 = 54
0x022A = 58
0x01D9 = 53
0x022B = 57
0x01DA = 60
0x022C = 65
0x01DE = 33
0x01E0 = 30
0x01E2 = 34
0x01E4 = 2
0x01E6 = 26
0x01E9 = 81
0x01EA = 6
0x01EB = 10
0x01EC = 26
0x01EE = -1
0x01EF = -1
0x01F0 = -1
0x01F4 = 32
0x01F5 = 9
0x01F6 = 34
0x0253 = 3
0x0254 = 3
0x01F8 = 2
0x01FC = -1
0x01FE = 5
0x0227 = 18
0x0228 = 18
0x0232 = 9
0x0233 = 11
0x0234 = 6
0x0230 = 12
0x022E = 71
0x027D = 62
0x0235 = -1
0x0239 = 11
0x022F = 5
0x0231 = 26
0x0237 = 2
0x0238 = 282
0x0236 = 10
0x023A = 4
0x023C = 6
0x023D = 6
0x023E = 8
0x023F = 2
0x0240 = -1
0x0241 = 6
0x0242 = -1
0x0243 = 6
0x0257 = 8
0x0244 = 6
0x0245 = 3
0x0246 = 4
0x0247 = 8
0x0248 = -1
0x0249 = 3
0x024A = 70
0x024B = 4
0x024C = 8
0x024D = 12
0x024E = 6
0x024F = 10
0x0250 = 3
0x0251 = 34
0x0252 = -1
0x0255 = 5
0x0256 = 5
0x0258 = 2
0x0259 = 3
0x025C = 4
0x025D = 6
0x025E = 4
0x025F = 6
0x0260 = 6
0x0261 = 11
0x0262 = 11
0x0263 = 11
0x0264 = 20
0x0265 = 20
0x0266 = 30
0x0267 = 4
0x0268 = 4
0x0269 = 4
0x026A = 4
0x026B = 4
0x026C = 4
0x026D = 4
0x026F = 2
0x0270 = 2
0x0271 = 40
0x0272 = 44
0x0273 = 30
0x0274 = 8
0x0275 = 37
0x0276 = -1
0x0277 = 84
0x0278 = 2
0x0279 = 2
0x027A = -1
0x027B = 14
0x027C = 60
0x027E = -1
0x027F = 8
0x0283 = 6
0x0287 = -1
0x0288 = 6
0x0289 = 8
0x0444 = -1
0x0445 = 6
0x028B = -1
0x028C = 46
0x028D = 34
0x028E = 4
0x028F = 6
0x0290 = 4
0x0291 = 4
0x0292 = 2
0x0293 = 70
0x0294 = 10
0x0295 = -1
0x0296 = -1
0x0297 = -1
0x0298 = 8
0x0299 = 6
0x029A = 27
0x029B = 80
0x029C = 66
0x029D = -1
0x029E = 11
0x029F = 3
0x02A2 = 8
0x02A5 = 8
0x02A6 = -1
0x02A7 = -1
0x02AA = 4
0x02AB = 36
0x02AC = 6
0x02AD = 8
0x02B0 = 85
0x02B8 = 22
0x02BB = 8
0x02B9 = 191
0x02BA = 11
0x02BC = 6
0x02B3 = 107
0x02B4 = 6
0x02B6 = 7
0x02B7 = 7
0x02C1 = -1
0x02C2 = -1
0x02C5 = 30
0x02C8 = 3
0x02C9 = 3
0x02C6 = 30
0x02C7 = 7
0x02CB = 65
0x02CC = 4
0x02CD = 71
0x02CE = 10
0x02CF = 6
0x02D5 = 2
0x02D0 = -1
0x02D1 = -1
0x02D2 = -1
0x02D3 = 4
0x02D4 = 29
0x02D6 = 6
0x02D7 = -1
0x02D8 = 10
0x02D9 = 10
0x02DA = 3
0x02DB = -1
0x02DC = -1
0x02DD = 32
0x02DE = 6
0x02DF = 36
0x02E0 = 34
0x02E1 = 33
0x02E2 = 8
0x02E3 = 10
0x02E4 = 6
0x02E5 = 5
0x02E6 = 6
0x02E7 = -1
0x02E8 = -1
0x02E9 = -1
0x02EA = -1
0x02EB = 13
0x02EC = 67
0x02ED = 59
0x02EE = 60
0x02EF = 8
0x02F0 = 10
0x02F1 = 2
0x02F2 = 2
0x035C = 2
0x035D = -1
0x035E = 2
0x03DD = 18
0x03DE = 18
0x0439 = 8
0x043D = 8
0x043E = -1
0x043F = 25
0x0440 = 10
0x0441 = 4
0x0442 = -1
0x0446 = 14
0x0448 = -1
0x0449 = 4
0x044A = 6
0x044B = 2
0x0447 = 2
0x07D7 = 8
0x07D8 = 8
0x07D9 = 268
0x07DA = 6
0x07DB = 8
0x07DC = 6
0x07DD = 54
0x07DE = 30
0x07DF = 54
0x07E0 = 58
0x07E1 = 15
0x07E2 = 8
0x07E3 = 6
0x07E6 = 8
0x07E5 = 4
0x07E8 = -1
0x07E7 = 32
0x07E9 = 5
0x07EA = 2
0x07EB = -1
0x07ED = 10
0x07EE = 6
0x07EF = 8
0x07F0 = 6
0x07F1 = 18
0x07F2 = 8
0x07F3 = 6
0x07F4 = 3
0x07F5 = 6
0x07F6 = 14
0x07F7 = -1
0x07F8 = -1
0x07F9 = -1
0x07FA = 8
0x07FB = 25
0x07FC = 10
0x07FD = -1
0x07FE = 26
0x0800 = -1
0x0801 = -1
0x0803 = 4
0x0804 = 14
0x0805 = -1
0x0806 = 2
0x0807 = 4
0x0808 = 14
0x0809 = 50
0x080A = 18
0x080B = 6
0x080C = 2
0x080D = 3
0x080E = 14
0x080F = 20
0x0810 = 3
0x0812 = 8
0x0813 = -1
0x0814 = 86
0x0816 = 6
0x0818 = -1
0x081A = 4
0x0824 = 6
0x081B = 10
0x081C = 10
0x081D = 22
0x081E = 8
0x081F = -1
0x0820 = 11
0x0821 = 2
0x0822 = 9
0x0823 = -1
0x0825 = -1
0x0826 = 4
0x0836 = -1
0x0837 = 3
0x0839 = 66
0x083A = 5
0x083B = 2
0x083D = 6
0x083E = 26
0x0840 = -1
0x0841 = 4
0x0827 = 6
0x0828 = 14
0x0829 = 12
0x082A = 10
0x082B = 6
0x082C = 10
0x0842 = 6
0x0843 = 6
0x0844 = 2
0x0845 = 6
0x0846 = 4
0x0847 = -1
0x0848 = -1
0x0849 = 12
0x084A = 2
0x084B = 19
0x084C = 10
0x084D = 10
0x084E = 5
0x084F = 6
0x0850 = 7
0x0855 = 6
0x0851 = -1
0x0852 = 2
0x0853 = -1
0x0854 = -1
0x0856 = -1
0x0857 = -1
0x0858 = -1
0x0859 = -1
0x08B1 = -1
0x082D = -1
0x08B2 = -1
0x08AF = 10
0x08B0 = 17
0x08B3 = -1
0x08B4 = 2
0x08B5 = 6
0x08B6 = 3
0x02F3 = -1
0x02F4 = 3
0x02F5 = 7
0x02F6 = 7
0x08B8 = 10
0x08B9 = 12
0x08BA = 10
0x08BB = 8
0x08BC = 10
0x08BD = 8
0x08BE = 14
0x08BF = 8
0x08C3 = 10
0x08C4 = 8
0x08C5 = 6
0x08C6 = 4
0x08C0 = -1
0x08C1 = 2
0x08C2 = 2
0x08C7 = -1
0x08C8 = 34
0x0923 = 2
0x08CA = -1
0x08CB = -1
0x097B = -1
0x08CC = 109
0x08CE = 2
0x08CF = 10
0x08D0 = 9
0x08D1 = 7
0x08D2 = 10
0x08D3 = 10
0x08D4 = 8
0x08D5 = -1
0x08D6 = 6
0x08D7 = 28
0x08D8 = 27
0x08D9 = 30
0x08DA = 26
0x08DB = 27
0x08DC = 26
0x08DD = 27
0x08DE = 27
0x08DF = 50
0x08E0 = 51
0x08E1 = 51
0x08E2 = 27
0x08E3 = 147
0x08E4 = 6
0x08FC = 30
0x08FD = 6
0x08FE = -1
0x08FF = 24
0x0900 = -1
0x0901 = -1
0x0902 = -1
0x0903 = -1
0x0904 = -1
0x0905 = -1
0x0906 = -1
0x0907 = 5
0x0908 = 5
0x090A = 26
0x090D = -1
0x090E = 2
0x0910 = 10
0x0911 = 30
0x0912 = 10
0x0913 = 30
0x0914 = -1
0x090F = -1
0x0915 = -1
0x0916 = 26
0x096B = 4
0x096C = 6
0x096D = -1
0x096E = -1
0x096F = 7
0x0971 = 6
0x0972 = -1
0x0973 = 7
0x0974 = 2
0x0975 = -1
0x0976 = -1
0x0977 = 14
0x0978 = 6
0x0979 = 50
0x097A = -1
0x097C = 4
0x097D = 288
0x097E = 12
0x097F = -1
0x0980 = 7
0x0981 = -1
0x0982 = 7
0x0983 = 29
0x0984 = 28
0x0985 = -1
0x0986 = 10
0x0987 = -1
0x0988 = 6
0x0989 = 2
0x098A = -1
0x098D = -1
0x098E = -1
0x098B = 2
0x098C = 4
0x098F = -1
0x0990 = 31
0x0991 = -1
0x0992 = -1
0x0993 = -1
0x0994 = -1
0x0995 = -1
0x0996 = -1
0x0997 = -1
0x0998 = 8
0x0999 = 11
0x099A = 9
0x099B = 8
0x099C = 6
0x099D = -1
0x099E = 12
0x099F = -1
0x09A0 = 6
0x09A1 = 2
0x09A2 = 6
0x09A3 = -1
0x09A4 = 18
0x09A5 = 7
0x09AB = 6
0x09A6 = 12
0x09A7 = 10
0x09A8 = 16
0x09A9 = 10
0x09AA = 16
0x09AC = -1
0x09AD = 10
0x09AE = 17
0x09AF = 4
0x09B0 = 8
0x09B1 = 4
0x09B2 = 8
0x09B3 = 4
0x09B4 = 6
0x09B5 = 2
0x09B6 = 6
0x09B7 = 4
0x09B8 = 6
0x09B9 = 4
0x09BA = 2
0x09BB = 6
0x09BC = 6
0x09BD = 2
0x09BE = 2
0x09BF = 4
0x09C1 = 10
0x09C2 = -1
0x09C3 = 8
0x09C4 = 8
0x09C5 = 1042
0x09C6 = -1
0x09C7 = 18
0x09C8 = -1
0x09C9 = -1
0x09CA = -1
0x09CB = 17
0x09CC = -1
0x09CD = 8
0x0966 = 102
0x09CF = -1
0x09D0 = -1
0x09D1 = 14
0x09D2 = -1
0x09D3 = -1
0x09D4 = 2
0x09D5 = -1
0x09D6 = -1
0x09D8 = 2
0x09D7 = -1
0x09D9 = 4
0x09DA = -1
0x09DB = -1
0x09DC = -1
0x09DD = -1
0x09DE = -1
0x09DF = 7
0x09E0 = -1
0x09E1 = 8
0x09E2 = 8
0x09E3 = 8
0x09E4 = 8
0x09E5 = 18
0x09E6 = 22
0x09E7 = 3
0x09E8 = 11
0x09E9 = 2
0x09EE = 11
0x09EF = 11
0x09F0 = -1
0x0A7D = -1
0x09F5 = 11
0x09F6 = 11
0x09EA = 11
0x09EB = -1
0x09F7 = 75
0x09EC = -1
0x09ED = 3
0x09F3 = 11
0x09F4 = 12
0x09F1 = 11
0x09F2 = 12
0x0A04 = 6
0x0A05 = 53
0x0A06 = 6
0x0A07 = 9
0x0A03 = 2
0x0A08 = 26
0x0A12 = 27
0x0A13 = 26
0x0A14 = 10
0x0A51 = 34
0x0A32 = 2
0x09F8 = -1
0x09F9 = 143
0x09FA = -1
0x09FB = -1
0x09FC = 6
0x09FD = -1
0x09FE = -1
0x09FF = -1
0x0A00 = 269
0x0A01 = 3
0x0A02 = 4
0x0A09 = 45
0x0A0A = 47
0x0A0B = 47
0x0A0C = 56
0x0A0D = -1
0x0A0F = -1
0x0A10 = -1
0x0A11 = -1
0x0A0E = 14
0x0A15 = 12
0x0A16 = 26
0x0A17 = 6
0x0A18 = 14
0x0A19 = 2
0x0A1A = 23
0x0A1B = 2
0x0A1C = -1
0x0A1D = 2
0x0A1E = 3
0x0A1F = 2
0x0A20 = 21
0x0A21 = 3
0x0A22 = 5
0x0A23 = -1
0x0A24 = 66
0x0A25 = 6
0x0A26 = 7
0x0A27 = 8
0x0A28 = 3
0x0A29 = 6
0x0A2A = 6
0x0A2B = 10
0x0A2C = 12
0x0A2D = -1
0x0A2E = 6
0x0A2F = 7
0x0A30 = 106
0x0A31 = -1
0x0A33 = 7
0x0A34 = 6
0x0A35 = 4
0x0A36 = 7
0x0A37 = 59
0x0A38 = 3
0x086A = 3
0x0AE2 = 7
0x0A39 = 36
0x0A3A = 12
0x0A3B = -1
0x0A3C = -1
0x0A3D = 18
0x0A3E = -1
0x0A3F = 9
0x0A40 = 11
0x0A41 = 18
0x0A42 = 43
0x0A43 = 85
0x0A44 = -1
0x0A46 = 14
0x0A47 = 3
0x0A48 = 2
0x0AFC = 16
0x0A49 = 20
0x0A4A = 6
0x0A4B = 22
0x0A4C = 28
0x0A8F = 2
0x0A90 = 3
0x0A4D = -1
0x0A79 = -1
0x0A4E = 4
0x0A70 = 2
0x0A4F = -1
0x0A50 = 4
0x0A52 = 20
0x0A53 = 10
0x0A54 = -1
0x0A55 = 2
0x0A56 = 6
0x0A57 = 6
0x0A58 = 8
0x0A59 = -1
0x0880 = 2
0x0A5B = 7
0x0876 = 18
0x0A5D = 6
0x0873 = 6
0x0A6A = 12
0x0A6B = -1
0x0931 = 7
0x0A6D = -1
0x0878 = -1
0x0A6F = -1
0x0A71 = -1
0x0A72 = 61
0x0A73 = 2
0x0A74 = 8
0x0A76 = 80
0x0A77 = 15
0x0A78 = 15
0x0A7B = -1
0x0A7C = -1
0x0A7E = -1
0x0A8C = 2
0x0A80 = 6
0x0A7F = -1
0x0A8D = -1
0x0A81 = 4
0x0A92 = -1
0x0A91 = -1
0x0A93 = 3
0x0A94 = 2
0x0A89 = 57
0x0B05 = 59
0x0A8A = 6
0x0A82 = 46
0x0A83 = 46
0x0A84 = 94
0x0A85 = 82
0x0A86 = -1
0x0A87 = -1
0x0A88 = 2
0x0A8B = 2
0x0A8E = 2
0x0A95 = 4
0x0A96 = 51
0x0A97 = 8
0x0A98 = 10
0x0A99 = 4
0x0A9A = 10
0x0A9B = -1
0x0A9C = 2
0x0ACE = 4
0x0A9D = 4
0x0A9E = 2
0x0A9F = 2
0x0AA0 = 2
0x0AA1 = 4
0x0AA2 = -1
0x0AA3 = 7
0x0AA4 = 2
0x0AA5 = -1
0x0AA6 = 36
0x0AA7 = 6
0x0AA8 = 5
0x0AA9 = -1
0x0AAA = -1
0x0AAB = -1
0x0AAC = 69
0x0AB1 = 14
0x0AAD = 51
0x0AAE = 2
0x0AAF = 6
0x0AB0 = 6
0x0ABA = 2
0x0ABB = 2
0x0AB2 = 7
0x0AB3 = 19
0x0AB4 = 4
0x0AB5 = 2
0x0AB6 = 6
0x0AB7 = 4
0x0AB8 = 2
0x0AB9 = 39
0x0ABC = -1
0x0ABD = 10
0x0ABE = -1
0x0ABF = -1
0x0AC0 = 26
0x0AC1 = 26
0x0AC2 = -1
0x0AC3 = 2
0x0AC4 = -1
0x0AC5 = 156
0x0AC6 = 156
0x0AC7 = 156
0x0AC8 = 2
0x0AC9 = -1
0x0ACA = 3
0x0ACB = 12
0x0ACC = 18
0x0ACD = 23
0x0ACF = 68
0x0AD0 = 11
0x0AD1 = -1
0x0AD2 = 30
0x0AD3 = -1
0x0AD4 = -1
0x0AD5 = 2
0x0AD6 = 2
0x0AD7 = 8
0x0AD8 = 8
0x0AD9 = -1
0x0ADA = 30
0x0ADB = -1
0x0ADC = 6
0x0ADD = 22
0x0ADE = 6
0x0ADF = 58
0x0AE0 = 30
0x0AE1 = 28
0x0AE4 = 89
0x0AE5 = -1
0x0AE3 = -1
0x0AE6 = 10
0x0AE7 = 38
0x0AE8 = 2
0x0AEC = 2
0x0AED = 2
0x0AEE = 2
0x0AE9 = 13
0x0AEF = 2
0x0AF0 = 10
0x0AF2 = 40
0x0AF3 = -1
0x0AF4 = 11
0x0AF5 = 3
0x0AF6 = 88
0x0AF7 = 32
0x0AF8 = 11
0x0AF9 = 6
0x0AFA = 54
0x0AFB = -1
0x0AFD = -1
0x0B0C = 155
0x0AFE = -1
0x0AFF = -1
0x0B00 = 8
0x0B01 = 56
0x0B02 = 26
0x0B03 = -1
0x0B04 = 80
0x0B06 = 53
0x0B07 = -1
0x0B08 = 27
0x0B09 = -1
0x0B0A = -1
0x0B0B = 4
0x0B0D = 10
0x0B0E = -1
0x0B0F = -1

[Packet_Keys]
1 = 0x53276CA0
2 = 0x654B52EB
3 = 0x571B46A8

[Shuffle_Packets]
0x091F = 0
0x0930 = 1
0x0955 = 2
0x0945 = 3
0x0957 = 4
0x0941 = 5
0x0881 = 6
0x0817 = 7
0x0920 = 8
0x086C = 9
0x0202 = 10
0x089D = 11
0x086B = 12
0x0948 = 13
0x0369 = 14
0x0969 = 15
0x07EC = 16
0x0964 = 17
0x0A6E = 18
0x0946 = 19
0x0934 = 20
0x0926 = 21
0x09CE = 22
0x0886 = 23
0x092F = 24
0x08A4 = 25
0x0360 = 26
0x089C = 27
0x0894 = 28

**Comment by Zexeta** [comment]
I wouldn't bother asking. You will be directed to a kRO shuffle file that isn't even the same length as last week's shuffle for restart. They will say to replace the right set of numbers with those that peek outputs which matches nothing that people are posting that partially works. If you take last week's shuffle and replace the right side it doesn't let you past char select. Just wait for the script n tutorial, cuz either everyone is explaining this very poorly or the other files output by the other script aren't correct.

**Comment by Fadreus** [comment]
>either everyone is explaining this very poorly or the other files output by the other script aren't correct.

"I failed in school. Either teacher bad in teaching or facilities not working."

It's not that everyone explaining is poor.
You basic knowledge is poor.

**Comment by Zexeta** [comment]
No one has explained how you get the left side of packets. Notice others posting their peek output asking what to do with them since it hasn't been properly explained. It's not from last week's shuffle, they aren't from the kro one Alison linked. The only post that remotely works is the one the guy posted with 200 packets and it even has problems. His set doesn't even match what Alison said to do either.

Excuse us for not spending years screwing with packets to instantly know how to set it up from bad explanations. I don't see you making an effort either aside from making snide comments and emoting at posts.

**Comment by c4c1n6kr3m1** [comment]
i said already, the left part is "original packets"
and it's not years, warpportal add the shuffle thing only about a year old
you can find it before that "starting" day ,  or learn it from serverType0.pm

**Comment by Fadreus** [comment]
Mlas aku mau cerita sama si tulul Zexeta ni.. :laughing: 
Udah dberi skrip, lgi mau diajari.

**Comment by alisonrag** [comment]
guys what is the problem to follow my tutorial?

i give you a step by step, a shuffle file to use as base.

**Comment by c4c1n6kr3m1** [comment]
may be because iRO has several shuffled packet additon? so the 28 or 29 or whatever is not enough
007D map_loaded
014D guild_check
014F guild_info_request
08C9 request_cashitems
i doubt they can login to map without this

**Comment by alisonrag** [comment]
left is peek generated and right is the base shuffle from kRO:
![image](https://user-images.githubusercontent.com/10372732/47255959-9b217980-d44e-11e8-8438-9cbc6151b9e3.png)

in the same order, just remove '0x' from peek file and replace the right packets in shuffle.txt

final shuffle file: (is in the right side)
![image](https://user-images.githubusercontent.com/10372732/47255997-24d14700-d44f-11e8-8df6-a08efd551b21.png)

**Comment by c4c1n6kr3m1** [comment]
iRO actor_action 0437
but we haven't added yet to iRO.pm

**Comment by ellenreval** [comment]
in peek there are 28 set of numbers,,, restart shuffle there are more than 30? what shoukd i replace?

**Comment by Fadreus** [comment]
I believe this is the root of the confusion:

- They asked how to connect for IRO RE:Start.
- Alison give the step, but they didn't get the shuffle.
- Gudoh add in-depth step, but they seems to cant understand it.
- Then Zelecktor asked for study purpose on shuffle.txt, then Alison answered.
- Next, for some reason these people thought that Alison give a step for iRO Re:Start & thought shuffle packets must have 29 line which is not because Alison double paste the comments on 2 thread which lead to this confusion.
- Then this thread started.

**Comment by alisonrag** [comment]
i'm checking the allanon tool output, a lot of packets is not shuffle

@c4c1n6kr3m1 
ty to show me that 007D is changed in iRO, we should add this to iRO.pm file because its not a realy shuffle (change every week) this is a new packet version

**Comment by alisonrag** [comment]
removing the regular shuffle list tje following packets remains:
```
007D 0365
0090 0A5A
00B8 0919
0126 0140
0127 0364
0140 07EC
0178 014D
014D 014F
014F 0815
0AF1 0821
08C9 0A69
09CE 07E4
0A68 083C
0A5A 022D
0A5C 0962
0A69 0A6C
0A6C 0811
0A6E 08C9
0A0D 0A8B
0AF1 0821
0AFD 0B0C
0438 0A68
035F 0437
083C 0838
07EC 0363
07E4 0281
0281 0436
0918 0940
0919 0918
093B 0361
0940 0202
0962 0A5C
`` 
it should know thet we have to add in iRO.pm

**Comment by Zexeta** [comment]
Yay people listened to what I said. I've been saying from the start going down n just replacing the right side with the output from peek did not work. It's not the correct amount.

**Comment by c4c1n6kr3m1** [comment]
@alisonrag , 007D  is shuffled regularly and these too, some are npc interaction
007D 0437
0090 091B
00B8 035F
0126 08A0
0127 0361
0140 0958
014D 0365
014F 083C
08C9 0923
09CE 0966
0A68 086A
0A5A 0880
0A5C 0876
0A69 0873
0A6C 0931
0A6E 0878
put these on iRO.pm is not a good idea

**Comment by Zexeta** [comment]
So c4 where did you get the additional shuffled packets from since apparently peek doesn't output them and no one else on this post knew they existed?

**Comment by alisonrag** [comment]
this is a protection that only exists in iRO =p

all other severs around the world use only 29 shuffle packets.

to get the packets you have to exclude sync packets and update in order

**Comment by Zexeta** [comment]
Yeah it's fine i understand, just a bit annoyed i spent hours trying to sort out why doing like you said didn't work then you have clowns like Fadreus making dumb comments when all i wanted to do was learn how to fix the bot on my own and no one was listening to me that it didnt match up.

**Comment by c4c1n6kr3m1** [comment]
@Zexeta you can see those addition if u really tried to compare
try this https://github.com/OpenKore/openkore/pull/2430

**Comment by Zexeta** [comment]
Seems to be working great so far, thank you :)

**Comment by Zexeta** [comment]
So is a script possible to create shuffle considering the additional packets iRO uses?

**Comment by alisonrag** [comment]
if the next update not add new packets you can continue using allanon script

**Comment by Zexeta** [comment]
Allanon's script didn't produce a shuffle last week.  I don't think new pixels were added that week either, they were added the previous week?

**Comment by alisonrag** [comment]
the problem in last week was new packets.
now c4c1n6kr3m1 added to recvpacket, so in next week if gravity not add others, the script will work

**Comment by Zexeta** [comment]
Ok thanks for all the help, appreciate it and i will try the script again.

**Comment by Zexeta** [comment]
The script did not produce a shuffle at all this week again.  It created the other files though.
